### DISCUSSION TITLE
a way to have two zones the same name for a single camera?

### DISCUSSION THREAD

**Discussion by spacebares** [blakeblackshear/frigate#4560] [Ask A Question] [answered]
Trying to setup a zone that is divided by another zone. Think of a yard with a driveway going threw in the middle. So on the camera when setting up the zone, you cant really have yard "split" down the middle. The coordinate system does not support this behavior.

So a solution is to make yard1, driveway, and  yard2 zones. I'm trying to avoid this, to make event sorting less confusing in events browser. 
Another solution is also to just make the yard zone cover driveway also. But now driveway events will show up in yard.

What can i do?

**Comment by NickM-27** [comment, marked as answer]
Should be able to make the zone one contiguous zone with a single line across the driveway. I'll see if I can get an example

**Reply by spacebares** [reply]
Something like this ?

![image](https://user-images.githubusercontent.com/57186372/204886962-1b798d84-ffd1-4d9e-ab70-b6b305a3447a.png)

**Reply by NickM-27** [reply]
yes, that should do it, the odds of the bottom center pixel being on that line are very unlikely I'd asy

**Reply by spacebares** [reply]
Although a bit rough to setup / modify later, this works.
